connectors/couchbase/translator-couchbase/pom.xml

```diff
@@ -29,6 +29,15 @@
         <dependency>
             <groupId>org.teiid.connectors</groupId>
             <artifactId>translator-jdbc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.teiid</groupId>
+            <artifactId>teiid-admin</artifactId>
+            <scope>compile</scope>
```

Comment:
Do we need this scope?

Reply:
The dependency yes, the scope no.  That was taken directly from the branch, but it will default to compile without that of course.